2023-04-16 00:45:01.407 WARNING (Recorder) [homeassistant.components.recorder.util] The system could not validate that the sqlite3 database at //config/home-assistant_v2.db was shutdown cleanly
2023-04-16 00:45:07.707 ERROR (MainThread) [homeassistant.config_entries] Error setting up entry Iris for spotify
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 383, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/usr/src/homeassistant/homeassistant/components/spotify/__init__.py", line 74, in async_setup_entry
    implementation = await async_get_config_entry_implementation(hass, entry)
  File "/usr/src/homeassistant/homeassistant/helpers/config_entry_oauth2_flow.py", line 401, in async_get_config_entry_implementation
    raise ValueError("Implementation not available")
ValueError: Implementation not available
2023-04-16 00:45:14.535 WARNING (MainThread) [homeassistant.config_entries] Config entry 'WLED' for wled integration not ready yet: Invalid response from API: Error occurred while communicating with WLED device at 192.168.15.102; Retrying in background
2023-04-16 00:45:14.554 WARNING (Recorder) [homeassistant.components.recorder.migration] Database is about to upgrade from schema version: 41 to: 35
2023-04-19 00:51:12.434 WARNING (Thread-2 (_thread_main)) [homeassistant.components.mqtt.client] Disconnected from MQTT server host.docker.internal:1883 (7)
